layouts/partials/fragments/list.html

```diff
@@ -0,0 +1,108 @@
+{{/*
+  This variable stores all the page bundles in the blog section. There are three
+  where functions achieving this. First where gets all the pages in blog
+  section, the second one removes current page from the mix and the last one
+  removes the section (.Site.Pages returns the current section as a page). */}}
+{{- $articles := where (where .Site.Pages "Section" (.Params.section | default .page.Section)) "Params.fragment" "in" (slice "content-single" "content-split") -}}
+{{- $bg := .Params.background | default "light" }}
+
+{{ "<!-- Blog -->" | safeHTML }}
+<section id="{{ .Name }}">
+  <div class="overlay container-fluid {{- printf " bg-%s" $bg -}}">
+    <div class="container py-5">
+      <l class="row">
+        <div class="col-12
+          {{- if or (eq $bg "secondary") (eq $bg "primary") -}}
+            {{- printf " text-%s" "dark" -}}
+          {{- else -}}
+            {{- printf " text-muted text-%s" "secondary" -}}
+          {{- end -}}
+        ">
+          {{- range first (.Params.article_count | default 10) $articles }}
```

Comment:
Could be shortened to just ".Params.count" I assume.

Reply:
Changed to `count` and also added default values in the example in b7caa1581b3a1ca548502229eef3e3efc828223e